.travis.yml

```diff
@@ -35,7 +35,9 @@ jobs:
         cp -r documentation/docs documentation/docs-tmp
         documentation/bin/createDocu.sh vars documentation/docs-tmp/steps
         docker run --rm -it -v ${TRAVIS_BUILD_DIR}:/docs -w /docs/documentation squidfunk/mkdocs-material:3.0.4 build --clean --verbose --strict
-    
+    - name: Consumer Tests for s4sdk pipeline
+      script: cd consumer-test && chmod +x runTests.sh && ./runTests.sh
```

Comment:
cd not strongly required.

```chmod +x consumer-test/runTests.sh && consumer-test/runTests.sh```

Reply:
But this will change what relative paths in the script refer to, right? I'd prefer it that way, but if there are issues with that, please raise them.